nemo_skills/pipeline/utils/eval.py

```diff
@@ -362,7 +362,10 @@ def prepare_eval_commands(
             for chunk_id in benchmark_chunk_ids:
                 job_benchmarks.add(benchmark)
 
-                generation_task = importlib.import_module(generation_module or benchmark_args.generation_module)
+                if os.sep in generation_module:
```

Comment:
the check needs to be for the combined thing `generation_module or benchmark_args.generation_module`. So perhaps assign it to variable first and then do if and reassignment to generation_task after import?